traffic_portal/app/src/common/api/JobService.js

```diff
@@ -31,9 +31,8 @@ var JobService = function($http, ENV) {
 	};
 
 	this.createJob = function(job) {
-		return $http.post(ENV.api['root'] + 'user/current/jobs', job).then(
+		return $http.post(ENV.api['root'] + 'jobs', job).then(
```

Comment:
i am unable to create a "job" directly from a delivery service. i.e. https://tp.domain.com/#!/delivery-services/2/jobs/new?type=HTTP and try to create a job. 

`deliveryService: cannot be blank., startTime: must be in the future`

Reply:
Oh, I updated the wrong controller. FormNewJobController isn't the controller for the new jobs form...